wagtail/wagtailadmin/static_src/wagtailadmin/scss/components/_modals.scss

```diff
@@ -114,3 +114,95 @@ $zindex-modal-background: 500;
         padding: 0 0 2em;
     }
 }
+
+
+
+.modal-tabs {
+    @extend %tab-nav;
+    margin-bottom: 0;
+}
+
+.modal-tabs--secondary {
+    margin-top: 0;
+    padding-top: 1.5rem;
+}
+
+
+    // TODO: Specificity arms race
+    .modal-tabs--secondary > .modal-tabs__item > a {
+        background: #D6D6D6;
+        color: #444;
+        border-top-color: #D6D6D6;
+
+        &:hover {
+          border-top-color: rgba(0, 0, 0, 0.4);
+        }
+    }
+
+    .modal-tabs--secondary > .modal-tabs__item.active > a {
+        background: #fff;
+        color: #222;
+        border-top-color: #333;
+    }
+
+
+// Much specificity required to add modifier.
+// TODO: refactor tabs to use lower specificity
+.modal-tabs--secondary,
+.modal-tabs__item--secondary.active > a {
+    background: #eee;
+}
+
+
+    .modal-tabs__item {
+        &:first-child {
+            margin-left: 0;
+        }
+
+        > a {
+            padding-left: 1.5rem;
+            padding-right: 1.5rem;
+            padding-bottom: .8125rem;
+            font-size: 13px;
+        }
+    }
+
+
+.modal-tabs__content {
+    padding-top: 1.5rem;
+}
+
+
+// =============================================================================
+// Inline form elements.
```

Comment:
Should this go in `_forms.scss`? It does not appear to be specifically related to modals.